### ISSUE TITLE
How to register helpers?

### ISSUE THREAD

**Issue by kraftwer1** [jfgodoy/broccoli-handlebars-commonjs#1]
How can I register helpers? I've tried it like that:

``` javascript
import Handlebars from "handlebars/runtime";

Handlebars.registerHelper("isStar", function( ... ));
```

``` handlebars
<div>{{#isStar}}*{{/isStar}}</div>
```

...but it looks like it doesn't find the helper.

**Comment by jfgodoy** [comment]
currently has not support for helpers, but I will add it today.

**Comment by kraftwer1** [comment]
:-)

**Comment by jfgodoy** [comment]
see the pull request #2 
I will wait your comments before merge and publish.

**Comment by kraftwer1** [comment]
Cool, looks great, thanks for your effort!

Question: Does the module at `runtimePath` already need to exist at the time when calling `broccoliHandlebars()`? Because I'm working with Webpack to access these templates and I have the following problem in my workflow:
1. Select `.js` and `.handlebars` using `new Funnel()`
2. Compile templates using `broccoliHandlebars()` <-- `handlebarsRuntime.js` is not a module, yet, and can therefore not be included
3. Transform `.js` into modules using `broccoliWebpackCached()` <-- `handlebarsRuntime.js` is now available

I think the best would be to be able to call `registerHelper()` at any time (also during runtime in the browser), but I guess that's not possible with the precompile-approach?

**Comment by jfgodoy** [comment]
the module at `runtimePath` is not require to compile the templates. I only use its path to add `require('path/to/runtimePath')` in the templates generated. So it is possible to define the helpers even in the browser at runtime.
Correct me if I wrong, but broccoliWebpackCached creates a bundle of modules that already exists, so at step 2,  you can reference your `handlebarsRuntime.js`

**Comment by kraftwer1** [comment]
Oh now I got it! The reason that in your example file you define all the helpers was leading me to think that they can only defined there. Plus, I thought that in your example, you were referencing to the file you created called `handlebarsRuntime` (and append the string `Runtime` to the file called `handlebars`) rather than the "real" Handlebars runtime.

Nice works man, thanks a lot!

**Comment by jfgodoy** [comment]
For the sake of clarity:
- templates compilation do not use helpers definitions.
- compiled templates need a handlebars runtime that include the helpers. So they call the require function to load the module defined at `runtimePath` option.
- If you want, at runtime you can require the module at `runtimePath` and define new helpers.

**Comment by jfgodoy** [comment]
I published as 1.1.0

**Comment by kraftwer1** [comment]
Thanks!
